lib/mumukit/core/object.rb

```diff
@@ -19,4 +19,8 @@ def defaulting(value)
   def struct(hash = {})
     hash.to_struct
   end
+
+  def filled?
```

Comment:
I would have defined `filled?` just as `present?`and made it `true` for `FalseClass`, but i agree this way is more explicit